src/test/java/com/wikia/webdriver/testcases/adstests/TestVUAP.java

```diff
@@ -9,18 +10,57 @@
 
 public class TestVUAP extends TemplateNoFirstLoad {
   private static final Dimension DESKTOP_SIZE = new Dimension(1920, 1080);
+  private static final String TOP_LEADERBOARD = "TOP_LEADERBOARD";
```

Comment:
Can we use existing selectors? https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java#L26